.github/workflows/CI-CD.yml

```
@@ -0,0 +1,47 @@
+name: IdlImporter CI/CD
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [windows-latest, ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        fetch-depth: 0
+
+    - name: Install .NET Core
+      uses: actions/setup-dotnet@77a48bbb85cd3809790223e895ba4b6cbc110fe8
+      with:
+        dotnet-version: 6.0.x
+
+    - name: Restore
+      run: dotnet restore
+
+    - name: Build
+      run: dotnet build
+
+    - name: Pack
+      run: dotnet pack IDLImporter --include-symbols --no-restore --no-build -o .
+
+    - name: Pack
+      run: dotnet pack IdlImpTool --include-symbols --no-restore --no-build -o .
+
+    - name: Publish to Nuget
+      run: dotnet nuget push *.nupkg -s https://api.nuget.org/v3/index.json -k ${{secrets.SILLSDEV_PUBLISH_NUGET_ORG}} --skip-duplicate
+      if: github.event_name == 'push' && matrix.os == 'ubuntu-latest'
+
+    - name: Publish Artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: NugetPackages
+        path: ./*.nupkg
+      if: github.event_name == 'pull_request'
```

It's unclear (and probably random) whether the Windows or Ubuntu build gets published. Maybe we should make this more explicit?

```suggestion
      if: github.event_name == 'pull_request' && matrix.os == 'ubuntu-latest'
```